po/it.po

```diff
@@ -0,0 +1,402 @@
+# Italian translation
+# Copyright (C) 2018 Bart Kessels
+# This file is distributed under the same license as the getit package.
+# Albano Battistella <albano_battistella@hotmail.com>, 2020.
+#
+msgid ""
+msgstr ""
+"Project-Id-Version: getit 4.0.6\n"
+"Report-Msgid-Bugs-To: bartkessels@bk-mail.com\n"
+"POT-Creation-Date: 2018-02-02 23:19+0100\n"
+"PO-Revision-Date: 2020-06-19 12:30+0100\n"
+"Last-Translator: Albano Battistella <albano_battistella@hotmail.com>\n"
+"Language-Team: Italian <LL@li.org>\n"
+"Language: it\n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+
+#: ../data/net.bartkessels.getit.appdata.xml.in:18
+msgid ""
+"GetIt's main focus is to make your life a lot easier when you have to test "
+"APIs or use somebody elses API. We know it can be hard to find a great, "
+"simple and open source application for this task that's not command line "
+"based."
+msgstr ""
+"L'obiettivo principale di GetIt è rendere la tua vita molto più semplice quando devi testare"
+"API o utilizzare qualche altra API. Sappiamo che può essere difficile trovare un ottima"
+"applicazione semplice e open source per questa attività che non sia basato sulla riga di" 
+comando."
```

Comment:
There's a quotation missing at the beginning of this line

Reply:
Hi,
done! :-)

Reply:
Thank you, you're PR has been merged